src/main/java/duke/userio/Ui.java

```diff
@@ -0,0 +1,166 @@
+package duke.userio;
+
+import duke.task.*;

```

Comment:
Perhaps you should avoid the use of asterisks to import the whole task package but import them individually when necessary instead?